documentation/peer-testing-1/heuristic-evaluation-experiment-instructions.md

```diff
@@ -7,11 +7,12 @@
 5. **Error prevention.** Technical problems should not occur during User operations. Eliminating or flagging actions that may result in errors are two possible means of achieving error prevention.
-6. **Recognition rather than recall.** Minimize cognitive load by maintaining task-relevant information within the display while users explore the interface. Human attention is limited and we are only capable of maintaining around five items in our short-term memory at one time. Due to the limitations of short-term memory, designers should ensure users can simply employ recognition instead of recalling information across parts of the dialogue. Recognizing something is always easier than recall because recognition involves perceiving cues that help us reach into our vast memory and allowing relevant information to surface. For example, we often find the format of multiple choice questions easier than short answer questions on a test because it only requires us to recognize the answer rather than recall it from our memory.
-7. **Flexibility and efficiency of use.** With increased use comes the demand for less interactions that allow faster navigation. This can be achieved by using abbreviations, function keys, hidden commands and macro facilities. Users should be able to customize or tailor the interface to suit their needs so that frequent actions can be achieved through more convenient means.
-8. **Aesthetic and minimalist design.** Keep clutter to a minimum. All unnecessary information competes for the user's limited attentional resources, which could inhibit user’s memory retrieval of relevant information. Therefore, the display must be reduced to only the necessary components for the current tasks, whilst providing clearly visible and unambiguous means of navigating to other content.
-9. **Help users recognize, diagnose and recover from errors.** Designers should assume users are unable to understand technical terminology, therefore, error messages should almost always be expressed in plain language to ensure nothing gets lost in translation.
-10. **Help and documentation.** Ideally, we want users to navigate the system without having to resort to documentation. However, depending on the type of solution, documentation may be necessary. When users require help, ensure it is easily located, specific to the task at hand and worded in a way that will guide them through the necessary steps towards a solution to the issue they are facing.
+6. **Recognition rather than recall.** UI designers should minimize the use of user's memory when they are guided to use the system. Recognizing something is always easier than recall because recognition involves perceiving cues that help us reach into our vast memory. For example, in some shopping website, there is a recently-viewed list to help users remind what they are interested in.
```

Comment:
The second sentence has been copied and pasted directly from https://www.interaction-design.org/literature/article/heuristic-evaluation-how-to-conduct-a-heuristic-evaluation?ref=seoptimer.com